src/components/clusterConfiguration/ReviewValidations.tsx

```diff
@@ -67,7 +67,10 @@ const FailingValidation: React.FC<{
 
   let fix;
   const step = findValidationFixStep({ id: validation.id, clusterGroup, hostGroup });
-  if (step) {
+  if (step === 'review') {
```

Comment:
Maybe we should just remove the 'all-hosts-are-ready-to-install' from `reviewStepValidationsMap`? User is not able to fix anything in the review step, so it makes no sense to include that validation there.

Doing so would highlight that validation as failed and link to a step would not show up as expected.

Reply:
But we need to disable `Install` in that case, don't we?